File: fo-frontend/src/services/certCommentService.js
import api from '@/api'

// 댓글 목록 조회
export function fetchComments(certificationId) {
	return api
		.get(`/certifications/${certificationId}/comments`)
		.then((res) => res.data)
}

// 댓글 등록
export function addComment(certificationId, content) {
	return api
		.post(`/certifications/${certificationId}/comments`, { content })
		.then((res) => res.data)
}

// 댓글 수정
export function updateComment(certificationId, commentId, content) {
	return api
		.put(`/certifications/${certificationId}/comments/${commentId}`, {
			content,
		})
		.then((res) => res.data)
}

// 댓글 삭제
export function deleteComment(certificationId, commentId) {
	return api
		.delete(`/certifications/${certificationId}/comments/${commentId}`)
		.then((res) => res.data)
}

File: fo-backend/src/main/java/com/hobby/challenge/fobackend/service/CertCommentService.java
package com.hobby.challenge.fobackend.service;

import java.util.List;

import com.hobby.challenge.fobackend.dto.CertCommentResponseDTO;

public interface CertCommentService {
	
    // 댓글 등록
    void addComment(int certId, int userId, String content);
	
    // 댓글 목록 조회
    List<CertCommentResponseDTO> getComments(int certificationId);

    // 댓글 수정
    void updateComment(int commentId, int userId, String content);

    // 댓글 삭제
    void deleteComment(int commentId, int userId);

}
